### PR TITLE
Bugfix message not identical exception

### PR DESCRIPTION
**Pull request by kbeker**
No issue for that. Problem appeared after fix of node_integration_tests

### CODE REVIEW

File: concent_api/core/validation.py [hunk 1]
```diff
@@ -127,20 +127,21 @@ def validate_all_messages_identical(golem_messages_list: List[message.Message])
...
-                    'The difference is on field {}: {} is not equal {}'.format(
-                        type(base_golem_message).__name__,
-                        i,
-                        slot,
-                        getattr(base_golem_message, slot),
-                        getattr(golem_message, slot),
-                    ),
-                    error_code=ErrorCode.MESSAGES_NOT_IDENTICAL,
-                )
+        if not base_golem_message.__eq__(golem_message):
```
Comment (rwrzesien) [inline concent_api/core/validation.py]: Was the problem here related with nested messages in TTC?

File: concent_api/core/tests/test_unit_authentication_utilities.py [hunk 2]
```diff
@@ -152,16 +149,11 @@ def test_that_validate_all_messages_identical_should_raise_http400_when_not_all_
...
-            message.TaskToCompute(
-                requestor_id = 1
-            ),
-            message.TaskToCompute(
-                requestor_id = 1
-            ),
-            message.TaskToCompute(
-                requestor_id = 1
-            ),
+            task_to_compute,
```
Comment (dybi) [inline concent_api/core/tests/test_unit_authentication_utilities.py]: please explicitly change `encrypted` param in one of the messages to be different

File: concent_api/core/validation.py (hunk 1 above)
Comment (kbeker) [reply concent_api/core/validation.py]: It was z problem with `encrypted` field in golem_messages. This field can be different and this method allows us to use method from GM and we will always know for sure is it equaled or not :)

Comment (dybi) [inline concent_api/core/validation.py]: 1) 
```suggestion
        if  base_golem_message != golem_message:
```
2) I would get rid of that "iteration" part that seeks which slot is different. But this is only my PoV - you decide ;)

File: concent_api/core/tests/test_unit_authentication_utilities.py (hunk 2 above)
Comment (kbeker) [reply concent_api/core/tests/test_unit_authentication_utilities.py]: Done

File: concent_api/core/validation.py (hunk 1 above)
Comment (kbeker) [reply concent_api/core/validation.py]: Fixed :)

**Pull request merged**